cookbooks/cdo-apps/templates/default/crontab.erb

```diff
@@ -96,6 +96,7 @@
       cronjob at:'0 */2 * * *', do:deploy_dir('bin', 'cron', 'teacher_applications_to_s3')
       cronjob at:'00 17 * * *', do:deploy_dir('bin', 'cron', 'stop_inactive_adhoc_instances')
       cronjob at:'0 10 * * *', do:deploy_dir('bin', 'cron', 'regional_partner_reporting')
+      cronjob at:'1 7 * * 6', do:deploy_dir('bin', 'cron', 'cleanup_workshop_attendance_codes')
```

Comment:
Just confirming we want this to run at 7:01am every Saturday (UTC)?

Reply:
Yes